machine_management/more-rhel-compute.adoc

```diff
@@ -26,9 +26,9 @@ include::modules/rhel-preparing-node.adoc[leveloffset=+1]
 include::modules/rhel-attaching-instance-aws.adoc[leveloffset=+1]
 
 .Additional resources
-* See xref:../installing/installing_aws/installing-aws-account.adoc#installation-aws-permissions_installing-aws-account[Required AWS permissions for IAM roles]
+* See xref:../installing/installing_aws/installing-aws-account.adoc#installation-aws-permissions-iam-roles_installing-aws-account[Required AWS permissions for IAM roles]
```

Comment:
Same here.  The other bullets on the page end with a period.  Please add one.

Reply:
Great catches! Fixed and pushed. PTAL